faunadb-scala/src/main/scala/faunadb/query/package.scala

```diff
@@ -2,6 +2,7 @@ package faunadb
 
 import com.fasterxml.jackson.annotation.JsonValue
 import faunadb.values._
+import java.time.Instant
```

Comment:
I didn't see any usage of this class, not in the context of this PR, is it a leftover?

Reply:
Whoops. `scalac` flags in the database prevent this from happening... perhaps they would help in this project, too?

Reply:
Sure, is it just set the flags or would it involve more configurations?

Reply:
That flag is just -Ywarn-unused-import.